core/serialize/serialize.h

```diff
@@ -6,7 +6,7 @@
 namespace sorbet::core::serialize {
 class Serializer {
 public:
-    static const u4 VERSION = 4;
+    static const u4 VERSION = 5;
```

Comment:
Does this version actually matter? I was under the impression that the on-disk cache was keyed by release SHA.

Reply:
Good question. This doesn't matter for KeyValueStore, but it _does_ matter for `--store-state`, which serializes GlobalState into a binary blob.

It prevents users from using a binary blob created with an earlier version of Sorbet.